internal/deprecations.go

```diff
@@ -97,6 +97,42 @@ func NormalizeClusterSpec(cluster *fdbv1beta2.FoundationDBCluster, options Depre
 		updateImageConfigs(&cluster.Spec, cluster.GetUseUnifiedImage())
 	}
 
+	if cluster.Spec.Buggify.CrashLoop != nil && len(cluster.Spec.Buggify.CrashLoop) > 0 {
+		crashLoopContainers := cluster.GetCrashLoopContainerProcessGroups()
+		if _, ok := crashLoopContainers[fdbv1beta2.MainContainerName]; !ok {
+			crashLoopContainers[fdbv1beta2.MainContainerName] = make(map[string]fdbv1beta2.None)
+			crashLoopContainers[fdbv1beta2.SidecarContainerName] = make(map[string]fdbv1beta2.None)
```

Comment:
```suggestion
			crashLoopContainers[fdbv1beta2.MainContainerName] = make(map[string]fdbv1beta2.None)
```

We don't have to add the sidecar container here.